src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -25,6 +26,14 @@
 
 (defmulti notification-component :type)
 
+;; TODO(rasom): should be removed as soon as all notifications types covered
+(defmethod notification-component :default
+  [{:keys [type]}]
+  [rn/view {:style {:width 300, :height 100}}
+   [rn/text
+    (gstring/format
+     "I exist just to avoid crashing for no reason. I'm sorry. Type %d" type)]])
```

Comment:
I think we don't need to tell Clojure devs that `defmethod` requires a default case, just like `case`, otherwise an exception is thrown if no match is found.

Reply:
It's just to emphasise without checking the code that this should be removed asap.

Reply:
It was funny to read it :smile: I just try to avoid comments explaining how something in the language works, so that's why I bothered you with this.